### PR TITLE
One true event, webcasts as array

### LINKED ISSUE #154: Change webcasts to string array of URLs, /events full events
As discussed on slack

### PR DESCRIPTION
**Pull request by fharding1**
# Goal
Close #154 

# Testing
Surely you're jest-ing?

### CODE REVIEW

File: internal/tba/tba.go [hunk 1]
```diff
@@ -124,11 +124,11 @@ func (s *Service) makeRequest(ctx context.Context, path string) (*http.Response,
 	return resp, nil
 }
 
-func webcastURL(webcastType store.WebcastType, channel string) (string, error) {
+func webcastURL(webcastType string, channel string) (string, error) {
```
[the remark this answers is not in the document]
Comment (fharding1) [reply internal/tba/tba.go]: :heart: thanks :heart_eyes:

[the remark this answers is not in the document]
Comment (brendanburkhart) [reply internal/tba/tba.go]: talking to a robot 🙄😂

[the remark this answers is not in the document]
Comment (fharding1) [reply internal/tba/tba.go]: bots need love too

**Comment by fharding1** [comment]
https://media.giphy.com/media/lae7QSMFxEkkE/giphy.gif

**Pull request merged**